tests/integration/dialog-extension/dialog_events.js

```diff
@@ -12,8 +12,33 @@ module( "dialog", {
 	}
 } );
 
-asyncTest( "Test option data-close-button", function() {
-	expect( 5 );
+asyncTest( "test option overlayTheme", function( assert ) {
+	$.testHelper.pageSequence( [
+
+		function() {
+			$( "#open-overlayTheme-test" ).click();
+		},
+
+		function() {
+			assert.deepEqual( $.mobile.activePage.attr( "id" ), "overlayTheme-test",
```

Comment:
Do not use `deepEqual` here its only for arrays, objects, regular expressions, dates and functions http://api.qunitjs.com/deepEqual/